wordpress-mobile/WordPress-FluxC-Android
# Use binary configure

This PR replaces the one-off encryption scheme in this project with the binary `configure` gradle plugin which allows us to avoid setting up Ruby in order to run `configure` using `fastlane`. 

**To Test CI Integration**
- Ensure automated tests pass

**To Test `apply`**
- Check out this PR locally
- Run `rm -rf .configure-files vendor && git reset --hard` to reset the repo to its default post-checkout state
- Run `./gradlew applyConfiguration`
- Note that the files are correctly decrypted in the `.configure-files` directory.

**To Test `update`**
- Check out this PR locally
- Run `rm -rf .configure-files vendor && git reset --hard` to reset the repo to its default post-checkout state
- Run `./gradlew updateConfiguration`
- Note that the `.configure` file has a new pinned hash corresponding to the latest mobile secrets commit
- Note that the `.enc` files have changed
